RecoBTag/FeatureTools/plugins/DeepBoostedJetTagInfoProducer.cc

```diff
@@ -324,6 +335,7 @@ void DeepBoostedJetTagInfoProducer::fillDescriptions(edm::ConfigurationDescripti
   desc.add<edm::InputTag>("secondary_vertices", edm::InputTag("inclusiveCandidateSecondaryVertices"));
   desc.add<edm::InputTag>("pf_candidates", edm::InputTag("particleFlow"));
   desc.add<edm::InputTag>("jets", edm::InputTag("ak8PFJetsPuppi"));
+  desc.addUntracked<edm::InputTag>("unsubjet_map", {});
```

Comment:
As discussed during ORP meeting Apr 2nd:

why is this untracked? Doesn't this change physics?

Reply:
ok, so just to be sure I understood:

Instead of changing the files: https://github.com/cms-sw/cmssw/blob/master/HLTrigger/Configuration/python/HLT_75e33/modules/hltPfDeepFlavourTagInfos_cfi.py and https://github.com/cms-sw/cmssw/blob/master/HLTrigger/Configuration/python/HLT_75e33/modules/hltPfDeepFlavourTagInfosModEta2p4_cfi.py

I change instead the file: https://github.com/cms-sw/cmssw/blob/master/HLTrigger/Configuration/python/customizeHLTforCMSSW.py

adding:

def customizeHLTfor44222(process):
    """
    Customisation for running HLT with the updated btag info producers from PR 44054
    """
    for type in ["DeepFlavourTagInfoProducer", "ParticleTransformerAK4TagInfoProducer", "DeepBoostedJetTagInfoProducer"]:
        for producer in producers_by_type(process, type):
            if not hasattr(producer, 'unsubjet_map'):
                producer.unsubjet_map = edm::InputTag("")
    return process

Reply:
> right?

no, if the change is only for modules in the folder `HLTrigger/Configuration/pythong/HLT_75e33` (which entails only phase-2 HLT menu, not yet supported on confDB) it's fine to change them by hand. The issue arises only for the modules employed in the standard Run3 menu (e.g. `HLTrigger/Configuration/python/HLT_GRun_cff.py`). If those are covered by the default value provided in the `fillDescriptions` (for all use-cases) there is no need of customization functions.

Reply:
ok, done in : https://github.com/cms-sw/cmssw/pull/44591

Reply:
14_0_4 parsing has those parameters present as UNTRACKED.
Thus also an entry in the customisation routine is needed to just remove the parameter by name.
Note it does not have to be present in the py config because fillDescriptions inserts the empty InputTag anyway. It is just that if it finds an untracked version with this PR, then it will fail!